package.json

```diff
@@ -219,6 +220,7 @@
 		"@wordpress/eslint-plugin": "^12.3.0",
 		"babel-loader": "^8.2.3",
 		"bunyan": "^1.8.15",
+		"canvas": "^2.9.3",
```

Comment:
What do we use this for?

Reply:
@noahtallen, I added this in https://github.com/Automattic/wp-calypso/pull/67013/commits/6c7a31f92090755d890f2e23e72834ac2e749a1f because the unit test associated with the banner was failing with the following: `TypeError: Cannot set property 'fillStyle' of null`

 I found that this is because `jest` isn't able to handle canvas by default (which is required by `lottie-web`), and requires a separate dependency for this purpose (similar issues found via Google [here](https://github.com/felippenardi/lottie-react-web/issues/21), [here](https://stackoverflow.com/questions/63219552/jest-import-statement-typeerror-cannot-set-property-fillstyle-of-null), and [here](https://robertmarshall.dev/blog/react-jest-tests-failing-using-lottie-js-solved/)). The [canvas dependency I added](https://www.npmjs.com/package/canvas) is an Automattic dependency.

Does this seem sound to you? Let me know if you have any further questions on this or if you have any concerns about this approach.

Reply:
Yep, that make sense to me! Thanks for clarifying :)